src/main/java/uk/gov/companieshouse/officerssearch/subdelta/kafka/Consumer.java

```diff
@@ -46,6 +49,7 @@ public Consumer(ServiceRouter router, MessageFlags messageFlags) {
     )
     public void consume(Message<ResourceChangedData> message) {
         try {
+            DataMapHolder.get().companyNumber(message.getPayload().getResourceId());
```

Comment:
The resource ID which is present on the ResourceChangedData object for an appointment refers to the appointment ID not the company number. The company number isn't relevant to officers search consumer either, we should be adding officer ID to the log map.

Reply:
Done, officer ID added to DataMapHolder within delete and upsert service instead of company number within the consumer.